public/static/js/version_admin.js

```diff
@@ -23,6 +23,9 @@ mciModule.controller('AdminOptionsCtrl', [
         // filtered set of builds
         return $scope.version.Builds.filter(buildFilter).map(
           function(x){
+            if (!x.Build.tasks) {
```

Comment:
Why not use x.Tasks instead? Will that be different from x.Build.Tasks? In this case that will be an empty array rather than undefined, so the line below shouldn't hit a runtime error

Reply:
Done.